crates/viewer/re_viewer_context/src/cache/video_stream_cache.rs

```diff
@@ -579,7 +580,9 @@ impl Cache for VideoStreamCache {
                     video_sample_buffers,
                 } = &mut *video_stream;
                 let video_data = video_renderer.data_descr_mut();
-                video_data.last_time_updated_samples = Some(Instant::now());
+                video_data.update_type = re_video::VideoUpdateType::Stream {
+                    last_time_updated_samples: Instant::now(),
+                };
```

Comment:
could maybe use a helper ctor?

```suggestion
                video_data.update_type = re_video::VideoUpdateType::new_stream();
```